app/src/main/java/com/sakaimobile/development/sakaiclient20/ui/adapters/AnnouncementsAdapter.java

```diff
@@ -42,7 +43,7 @@
     // mapping siteIdToCourse, needed to get subject code and course title
     private HashMap<String, Course> siteIdToCourse;
```

Comment:
Can't this just be a String to String map for siteId -> course name if you only need the course title?

Reply:
I also need the subject code of the course so that I can show the right icon